packages/logging/src/servicies/logger/logger.service.ts

```diff
@@ -35,7 +35,7 @@ export class LoggerService implements Logger {
       ...inputOptions
     };
 
-    if (typeof window === "undefined" && config.STD_OUT_LOG_FORMAT === "pretty") {
+    if (typeof window === "undefined" && config.STD_OUT_LOG_FORMAT === "pretty" && config.NODE_ENV !== "production") {
```

Comment:
this class shouldn't depend on NODE_ENV, only on env vars specific to it and direct config passed to the constructor.
```suggestion
    if (typeof window === "undefined" && config.STD_OUT_LOG_FORMAT === "pretty") {
```